webui/react/src/components/PageHeaderFoldable.tsx

```diff
@@ -16,15 +16,21 @@ export interface Option {
   label: React.ReactNode;
   onClick?: (ev: React.MouseEvent) => void;
   tooltip?: string;
+  content?: React.ReactNode;
 }
 
+export interface HeaderOption {
+  content: React.ReactNode;
+  key: string;
+  menuOptions: Option[];
+}
```

Comment:
Can we consolidate `menuOptions` into `content`, would that make the code at this component easier?

Reply:
@gt2345 I dont think sp, this was done because in this specific case we want to have one button, the continue trial button, that turns into two separate menu line options when hidden.  So we want to be able to map one specific react node to multiple options. Let me know if that makes sense or if you still think there is a better way to do this.

Reply:
I see, right, this is more complicated than what I had in mind, thanks for your explanation